tst/schedulers/test_searchers.py

```diff
@@ -66,7 +66,7 @@
 
 # Does not contain ASHABORE, because >10 secs on CI
 # TODO: Dig more, why is ASHABORE more expensive than BORE here?
-@pytest.mark.timeout(10)
+@pytest.mark.timeout(12)
```

Comment:
Looks like writing the files to disk makes the test run longer? Is this with intermediate checkpoints enabled, or is this just writing final output to disk?

Reply:
I am not sure. I just increased this because I had this test failing. I do not think this has anything to do with the change here, to be honest. The change here is not activated in this test, right? Final results are only written when you activate the feature.

Reply:
We can bump the limit, but if it was passing reliably before this PR, and starts timing out with this PR, then it would be useful to know why.

Not necessarily a blocker but something interesting